GherkinSpec.DogFoodTests/GherkinSpec.DogFoodTests.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@DaveAurionix if we want to keep these tests I'll update the build yml to run them

Reply:
Removed as discussed